exasol/python_extension_common/deployment/language_container_deployer_cli.py

```diff
@@ -82,32 +76,73 @@ def clear_formatters(self):
 slc_parameter_formatters = _ParameterFormatters()
 
 
+# This text will be displayed instead of the actual value, if found in an environment
+# variable, in a prompt.
+SECRET_DISPLAY = '***'
+
+
+class SecretParams(Enum):
+    """
+    This enum serves as a definition of confidential parameters which values should not be
+    displayed in the console, unless the user types them in the command line.
+
+    The enum name is also the name of the environment variable where the correspondent
+    secret value can be stored.
+
+    The enum value is also the name of the cli parameter.
+    """
+    DB_PASSWORD = 'db-pass'
+    BUCKETFS_PASSWORD = 'bucketfs-password'
+    SAAS_ACCOUNT_ID = 'saas-account-id'
+    SAAS_DATABASE_ID = 'saas-database-id'
+    SAAS_TOKEN = 'saas-token'
+
+
+def secret_callback(ctx: click.Context, param: click.Option, value: Any):
+    """
+    Here we try to get the secret parameter value from an environment variable.
+    The reason for doing this in the callback instead of using a callable default is
+    that we don't want the default to be displayed in the prompt. There seems to
+    be no way of altering this behaviour.
```

Comment:
```suggestion
    Retrieve secret parameter from the associated environment variable. 
    Other than a CLI option with callable default, this implementation ensures 
    confidentiality by being excluded from the CLI options help.
```

Reply:
Hmmm. My version may sound less like technical writing, but I think it's a more straightforward explanation of the issue.